### ISSUE TITLE
Not loading using stealJS

### ISSUE THREAD

**Issue by RALifeCoach** [matthewp/fs#11]
I have an app that I am trying to run unit tests. The app uses CanJS and StealJS. I am using karma and jasmine. I can get everything to work to run a single test file, but I want it to work using multiple test files. To that end I tried including fs-web so that I could dynamically load all the unit test files. The module was crashing so I simplified things to this point.
```javascript
import test1 from "tests/scripts/social/base-test";
import test2 from "tests/scripts/utils/path-test";
import fs from 'fs-web';
```
The file without the fs-web runs. As soon as I include the line to import fs-web, it throws an exception. (I also tried import * as fs from 'fs-web';) This exception happens when I run the test using karma runner. Everything looks better when it runs under node. The error I am getting is AMD module ... did not define.

'error', TypeError{line: 4342, sourceURL: 'http://localhost:9876/base/node_modules/steal/steal.js?d527490f310950704a06d564a96d5751d32ae8cf', stack: '	at instantiate (http://localhost:9876/base/node_modules/steal/steal.js?d527490f310950704a06d564a96d5751d32ae8cf:4342:77)
	at instantiate (http://localhost:9876/base/node_modules/steal/steal.js?d527490f310950704a06d564a96d5751d32ae8cf:4663:36)
	at instantiate (http://localhost:9876/base/node_modules/steal/steal.js?d527490f310950704a06d564a96d5751d32ae8cf:5751:34)
	at instantiate (http://localhost:9876/base/node_modules/steal/steal.js?d527490f310950704a06d564a96d5751d32ae8cf:5201:32)...

**Comment by matthewp** [comment]
Hey, I just released 1.0.1 which fixes the global. Your stealconfig should contain all of this:

``` js
steal.config({
  map: {
    "fs-web/fs-web": "fs-web"
  },
  paths: {
    "fs-web": "node_modules/fs-web/dist/fs.js"
  },
  meta: {
    "fs-web": {
      "format": "global",
      "exports": "fsWeb"
    }
  }
});
```

**Comment by RALifeCoach** [comment]
[image: Inline image 1]

On Wed, Jan 27, 2016 at 11:54 AM, Matthew Phillips <notifications@github.com

> wrote:
> 
> Hey, I just released 1.0.1 which fixes the global. Your stealconfig should
> contain all of this:
> 
> steal.config({
>   map: {
>     "fs-web/fs-web": "fs-web"
>   },
>   paths: {
>     "fs-web": "node_modules/fs-web/dist/fs.js"
>   },
>   meta: {
>     "fs-web": {
>       "format": "global",
>       "exports": "fsWeb"
>     }
>   }
> });
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/matthewp/fs/issues/11#issuecomment-175741590.

## 

Christopher Oliphant
+1 647 972 4549
